api_fhir_r4/configurations/R4ClaimConfig.py

```diff
@@ -57,25 +60,42 @@ def get_fhir_claim_item_status_rejected_code(cls):
 
     @classmethod
     def get_fhir_claim_item_general_adjudication_code(cls):
-        return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_item_general_adjudication_code', "general")
+        return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_item_general_adjudication_code',
+                                                                    "general")
 
     @classmethod
     def get_fhir_claim_item_rejected_reason_adjudication_code(cls):
         return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_item_rejected_reason_adjudication_code',
-                                                         "rejected_reason")
+                                                                    "rejected_reason")
 
     @classmethod
     def get_fhir_claim_organization_code(cls):
-        return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_organization_code', "openIMIS")  # has to be updated when 'Organization' is created
+        return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_organization_code',
+                                                                    "openIMIS")  # has to be updated when 'Organization' is created
 
     @classmethod
     def get_fhir_claim_attachment_code(cls):
         return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_attachment_code', "attachment")
 
     @classmethod
-    def get_fhir_claim_attachment_system(cls):
-        return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_information_category_system',
-                                                         "http://terminology.hl7.org/CodeSystem/claiminformationcategory")
+    def get_fhir_claim_visit_type_system(cls):
+        return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_visit_type_system',
+                                                                    'https://openimis.github.io/openimis_fhir_r4_ig/CodeSystem-claim-visit-type.html')
+
+    @classmethod
+    def get_fhir_claim_supporting_info_system(cls):
+        return cls.get_config_attribute("R4_fhir_claim_config").get('fhir_claim_supporting_info_category_system',
+                                                                    'https://openimis.github.io/openimis_fhir_r4_ig/CodeSystem/claim-supporting-info-category')
```

Comment:
Please add the base URL 'https://openimis.github.io/openimis_fhir_r4_ig' in a constant variable. It might change when we finish the IG so it will be easier to replace. Check other places where to update. Thanks.

Reply:
I made all system strings in `claimConverter` relative to base system URL from `GeneralConfiguration`